RxCocoa/iOS/UITableView+Rx.swift

```diff
@@ -168,15 +168,6 @@ extension Reactive where Base: UITableView {
 extension UITableView {
  
     /**
-    Factory method that enables subclasses to implement their own `delegate`.
-    
-    - returns: Instance of delegate proxy that wraps `delegate`.
-    */
-    public override func createRxDelegateProxy() -> RxScrollViewDelegateProxy {
```

Comment:
Let's try to find some better solution for this, I'll also try to figure out some better way.